src/main/java/uk/gov/hmcts/cmc/claimstore/config/HttpClientConfiguration.java

```diff
@@ -5,31 +5,46 @@
 import org.apache.http.HttpRequestInterceptor;
 import org.apache.http.HttpResponseInterceptor;
 import org.apache.http.client.config.RequestConfig;
+import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
 import uk.gov.hmcts.reform.logging.httpcomponents.OutboundRequestIdSettingInterceptor;
 import uk.gov.hmcts.reform.logging.httpcomponents.OutboundRequestLoggingInterceptor;
 
 @Configuration
-public class FeignClientConfiguration {
+public class HttpClientConfiguration {
 
     @Bean
     public Client getFeignHttpClient() {
+        return new ApacheHttpClient(getApacheHttpClient());
+    }
+
+    @Bean
+    public RestTemplate restTemplate() {
+        RestTemplate restTemplate = new RestTemplate();
+        restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory(getApacheHttpClient()));
+        return restTemplate;
+    }
+
+    private CloseableHttpClient getApacheHttpClient() {
```

Comment:
I don't think we should care that it's an apache class.

Reply:
Done 👍